config/default/core.entity_view_display.node.hs_basic_page.default.yml

```diff
@@ -20,8 +20,8 @@ third_party_settings:
         layout_id: three_column
         layout_settings:
           label: ''
-          section_width: ''
-          region_widths: ''
+          section_width: hs-full-width
+          region_widths: center
```

Comment:
This is so simple!

It looks like it'll apply to new sites going forward but non necessarily existing sites. And that's probably fine. 👍

Are we pretty sure this won't negatively impact sites on the `su_humsci_theme`?

Reply:
I showed it to Mike, and he's ok with it! Yes, will affect all sites, but so did our value change to 'hs-full-width' so this will be good.

But I think the center value is probably an unnecessary change since we didn't do anything with that, or change that value.

Reply:
Yeah this is all we changed last time: https://github.com/SU-HSDO/suhumsci/commit/fd1e876816ba7538c6e037a9e57e2d4fa7a84131#diff-5d94d30d84cf534b949034c88ad89648

Reply:
Cool. I was looking into what we can do for existing sites, and it looks like it's just a matter of browsing to `/admin/structure/types/manage/hs_basic_page/display/default/layout` and saving the top section.

In other words, we make changes to this "default" layout even when other individual `basic_page`s  have diverged from the default.

So fixing this for existing sites should just be a one-time, manual configuration change. This may be worth showing during demo, just so their team knows how to get here. 👍

Reply:
This is talking about the basic page layout for ALL basic pages.

Reply:
one offs may act different